### ISSUE TITLE
[BUG] [BUN] Post request body reading in production build hangs forever using Bun

### ISSUE THREAD

**Issue by itsyoboieltr** [nksaraf/vinxi#95]
Reproduction: [solidstart-post-repro](https://github.com/itsyoboieltr/solidstart-post-repro)

Using the [bun](https://bun.sh) adapter, only in `production` builds (during dev this works), getting the body of a post request will hang forever.

```ts
export const POST = async (event: APIEvent) => {
  console.log(await event.request.json());
  return new Response('ok');
};
```

Possible ideas what could be wrong:
- [nitro](https://github.com/unjs/nitro) bun preset
- [h3](https://github.com/unjs/h3) handler

**Comment by edivados** [comment]
This seems to be fixed in `0.1.3`. Running the example with vinxi `0.1.3` in prod worked for me.

References:
- https://github.com/nksaraf/vinxi/pull/123
- https://github.com/solidjs/solid-start/pull/1255

**Comment by itsyoboieltr** [comment]
@edivados it does work! Thank you for your work guys :)
